docs/share_logs.md

```diff
@@ -0,0 +1,52 @@
+## Share your logs with us to help improve Auto-GPT
+
+Do you notice weird behavior with your agent? Do you have an interesting use case? Do you have a bug you want to report?
+Follow the steps below to enable your logs and upload them. You can include these logs when making an issue report or discussing an issue with us.
+
+### Enable Debug Logs
+Activity and error logs are located in the `./output/logs`
```

Comment:
This is incorrect, same as the current usage documentation :x

Reply:
@NeonN3mesis can you please take a look here to fix the issue @Pwuts is hinting at? I'd be happy to fix it myself but I'm not sure what part is outdated exactly

Reply:
I'm on it

Reply:
I made a new PR on your branch @mlejva

Reply:
I merged it